### PR TITLE
Add internal link to header

### LINKED ISSUE #965: Headlines not supporting html code after deploy
In the [Network](https://okfn.org/en/network/) page, I've been using HTML coding in the Headline plugin to create anchor links and help users browse through the page. Like in the example below:

Screenshot 2023-08-23 at 10 49 09

Now, as you can see, this code is being displayed. Before it became "hidden" and worked pretty well. Could you pls check why this is happening and revert the changes that are preventing HTML to be supported?

(I think it might be related to markdown)

### LINKED ISSUE #966: Bug on OKimage plugin layout after deploy
One of the correct layouts for the OKImage plugin, when I want to use it to feature content, is the following (as is on the [homepage](https://okfn.org/en/)):

Screenshot 2023-08-23 at 10 56 09

But for some reason after the deployment it's been displayed like a normal picture. The text is now shown as a caption, and the tag field completely disappeared. This example is from the [Tech Services](https://okfn.org/en/our-services/tech-services/) page.

Screenshot 2023-08-23 at 10 54 13

I tried to put the plugin inside a Grid Column, but it didn't work. Interestingly, the plugin is still working on the Homepage, but I assume it's because the homepage layout renders differently.

We need to revert it asap, pls.

### PR DESCRIPTION
**Pull request by shashigharti**
- fixes #965

### CODE REVIEW

**Comment by shashigharti** [comment]
@pdelboca it is ready for review. Thank you!

File: foundation/okfplugins/heading/models.py [hunk 1]
```diff
@@ -20,6 +20,8 @@ class Heading(CMSPlugin):
         max_length=6, choices=HEADINGS_CHOICES, default="h1"
     )
     highlighted = models.BooleanField(default=False)
+    is_anchor = models.BooleanField(default=False)
+    anchor_id = models.CharField(max_length=200, blank=True)
```
Comment (pdelboca) [inline foundation/okfplugins/heading/models.py:24]: Do we need two fields? Is it possible to just check if `anchor_id` has a value?

File: foundation/okfplugins/heading/templates/heading_plugin.html [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
 <div class="headline -tighten-1 text-center mb-20">
-  <h1 class="col-span-full px-3 py-5 font-{% if instance.highlighted %}mono{% else %}bold{% endif %} text-{{ instance.alignment }} text-{{ instance.heading_type }}">{{ instance.title | markdown }}</h1>
+  <h1 class="col-span-full px-3 py-5 font-{% if instance.highlighted %}mono{% else %}bold{% endif %} text-{{ instance.alignment }} text-{{ instance.heading_type }}">
+    {% if instance.is_anchor %}
+    <a id="{{ instance.anchor_id }}">
+    {% endif %}
+      {{ instance.title | markdown }}
+    {% if instance.is_anchor %}
+    </a>
+    {% endif %}
```
Comment (pdelboca) [inline foundation/okfplugins/heading/templates/heading_plugin.html:11]: I think in this case is better, for readability, to use a syntaxis like this:
```
{% if instance.anchor_id %}
  <a id="{{ instance.anchor_id }}">{{ instance.title | markdown }}</a>
{% else %}
  {{ instance.title | markdown }}
{% endif %}
```

Even when we are repeating `{{ instance.title | markdown }}` it is betther to not spllit the `<a>` element.

**Review by pdelboca — APPROVED** [review]

**Comment by pdelboca** [comment]
@shashigharti I approved it but checkout my comments on the implementation :)

File: foundation/okfplugins/heading/models.py (hunk 1 above)
Comment (shashigharti) [reply foundation/okfplugins/heading/models.py]: No. yes! I will remove the boolean field.

File: foundation/okfplugins/heading/templates/heading_plugin.html (hunk 2 above)
Comment (shashigharti) [reply foundation/okfplugins/heading/templates/heading_plugin.html]: yes! this is better.

**Pull request merged**